apps/api/src/admin/participant_manager.py

```diff
@@ -15,7 +15,7 @@ class Participant(UserRecord):
 
     first_name: str
     last_name: str
-    status: Union[Status, Decision]
+    status: Optional[Union[Status, Decision]] = None
```

Comment:
Question: in what cases would a participant not have a status?

Reply:
I had to alter this to fit with non-hackers because there are some non-hackers who haven't signed the waiver yet and therefore don't have a status.

Reply:
Hmm, do you think you could manually assign a status to those who don't? Don't wanna mess too much with entire system right now, so maybe something like `REVIEWED`, even though it technically doesn't make sense?

Reply:
How about `WAIVER_UNSIGNED`?

Reply:
Oh never mind, that's not part of `Status`. I'll give them `REVIEWED`.

Reply:
Oh i meant the database records, but I suppose this also technically works.